akka-projection-cassandra/src/test/scala/akka/projection/cassandra/CassandraProjectionSpec.scala

```diff
@@ -623,26 +623,19 @@ class CassandraProjectionSpec
       val entityId = UUID.randomUUID().toString
       val projectionId = genRandomProjectionId()
 
+      type Offset = Long
+
       val flowHandler =
-        FlowWithContext[Envelope, Envelope]
+        FlowWithContext[Envelope, ProjectionContext[Offset, Envelope]]
```

Comment:
@ignasi35 maybe we could make the public API just `ProjectionContext` without any type parameters. Users should not care about the content of the context, just pass it along. Then internally we can cast it to a `ProjectionContextImpl` with the needed fields. I don't mind adding a timestamp there, and if more context propagation is needed we have a place to attach more things with bytecode manipulation.